governance/polls/Ratification Poll - Task Forces (MIP75) - June 13, 2022.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ratification Poll for Task Forces (MIP75) - June 13, 2022.
+summary: MIP75 proposes the formalization of Task Forces. These groups will be responsible for coordinating high-level business decisions on behalf of the DAO. They may also work with an associated discretionary fund to realize business objectives.
+discussion_link: https://forum.makerdao.com/t/mip75-task-forces/15093
+vote_type: Plurality Voting
+categories:
+   - MIPs
+   - High Impact
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-06-13T16:00:00
+end_date: 2022-06-27T16:00:00
+---
+# Ratification Poll for Task Forces (MIP75) - June 13, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, June 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP75 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP75 proposes the formalization of Task Forces. These groups are designed to directly interface with the delegates to coordinate and agree upon terms in advance to provide more autonomy within the Core Units. Task Forces can be formed by anyone, but the ratification process needs to be put to a vote due to discretionary fund requests being attached. Task Forces are free to add or remove members by their own consensus rules.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/626570f9e3e20708f6223a463b6e839259e91c76)
```

Comment:
Use this URL: https://github.com/makerdao/mips/blob/626570f9e3e20708f6223a463b6e839259e91c76/MIP75/MIP75.md